platform/ontology/api.go

```diff
@@ -0,0 +1,117 @@
+package ontology
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	"github.com/trustwallet/blockatlas/coin"
+	"github.com/trustwallet/blockatlas/models"
+	"github.com/trustwallet/blockatlas/util"
+	"net/http"
+	"strings"
+)
+
+var client = Client{
+	HTTPClient: http.DefaultClient,
+}
+
+const GovernanceContract = "AFmseVrdL9f9oyCzZefL9tG6UbviEH9ugK"
+
+// Setup registers the Ontology route
+func Setup(router gin.IRouter) {
+	router.Use(util.RequireConfig("ontology.api"))
+	router.Use(func(c *gin.Context) {
+		client.BaseURL = viper.GetString("ontology.api")
+		c.Next()
+	})
+	router.GET("/:address", getTransactions)
+}
+
+func getTransactions(c *gin.Context) {
+	var token = c.DefaultQuery("token", "ont")
+	var address = c.Param("address")
+
+	txPage, error := client.GetTxsOfAddress(address, token, 1)
+
+	if error != nil {
+		logrus.WithError(error).
+			Errorf("Ontology: Failed to get transactions for %s, token %s", address, token)
+	}
+
+	var txs []models.Tx
+	for _, tx := range txPage.Result.TxnList {
+		if txNormalized, ok := Normalize(&tx, token); ok == true {
+			txs = append(txs, txNormalized)
+		}
+	}
+
+	page := models.Response(txs)
+	page.Sort()
+	c.JSON(http.StatusOK, &page)
+}
+
+func Normalize(srcTx *Tx, assetName string) (tx models.Tx, ok bool) {
+
+	transfer := srcTx.TransferList[0]
+	fee := util.DecimalExp(srcTx.Fee, 9)
+	var status string
+	if srcTx.ConfirmFlag == 1 {
+		status = models.StatusCompleted
+	} else {
+		status = models.StatusFailed
+	}
+
+	tx = models.Tx{
+		ID: srcTx.TxnHash,
+		Coin: coin.ONT,
+		Fee: models.Amount(fee),
+		Date:  srcTx.TxnTime,
+		Block: srcTx.Height,
+		Status: status,
+	}
+
+	// Condition for transfer ONT
+	if assetName == "ont" {
```

Comment:
where would you get `ont` value from? user will be requesting `/transactions/:address`

Reply:
it's here 
https://github.com/TrustWallet/blockatlas/blob/1cef2ded64f0d831ff34a433e4986c629d332b98/platform/ontology/api.go#L31

Reply:
I see now, can you have it as constant variables? because it's reused

Reply:
Done